int64 size  # 有效点数量
int64 frame_id
float32 coord_x
float32 coord_y
float32 coord_z
float32 camToWorldx # 当前相机位姿
float32 camToWorldy
float32 camToWorldz
float32 camToWorldRx
float32 camToWorldRy
float32 camToWorldRz
float32 camToWorldRw
float32[] x
float32[] y
float32[] z
float32 scale